store/sqlstore/post_store.go

```diff
@@ -1358,7 +1389,7 @@ func (s *SqlPostStore) GetRepliesForExport(parentId string) ([]*model.ReplyForEx
 			INNER JOIN
 				Users ON Posts.UserId = Users.Id
 			WHERE
-				Posts.ParentId = :ParentId
+				Posts.RootId = :ParentId
```

Comment:
Is it worth propagating `parentId` backwards as a `rootId` argument in the caller code?

Reply:
Sure thing!